### PR TITLE
Adding 213 support

### PR DESCRIPTION
**Pull request by manodnyab**
## Description

Adding support for 2021.3

 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src-211+/software.aws.toolkits.jetbrains/services/ssm/SsmPlugin.kt [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        return FourPartVersion.parse(processOutput.stdout.trim())
+    }
+
+    override fun downloadVersion(version: FourPartVersion, destinationDir: Path, indicator: ProgressIndicator?): Path {
+        // TODO: Move to CpuArch, FIX_WHEN_MIN_IS_211
+        val downloadUrl = when {
+            SystemInfo.isWindows -> windowsUrl(version)
+            SystemInfo.isMac -> macUrl(version)
+            SystemInfo.isLinux && isUbuntu && isArm64() -> ubuntuArm64Url(version)
+            SystemInfo.isLinux && isUbuntu && CpuArch.isIntel64() -> ubuntuI64Url(version)
```
Comment (kiiadi) [inline jetbrains-core/src-211+/software.aws.toolkits.jetbrains/services/ssm/SsmPlugin.kt:54]: for this - lets not copy the *whole* file, lets create a seprate file, that has *just* the required compatibility functions in it.

**Review by kiiadi — CHANGES_REQUESTED** [review]

File: jetbrains-core/src-211+/software.aws.toolkits.jetbrains/services/ssm/SsmPlugin.kt (hunk 1 above)
Comment (manodnyab) [reply jetbrains-core/src-211+/software.aws.toolkits.jetbrains/services/ssm/SsmPlugin.kt]: This was merged with : https://github.com/aws/aws-toolkit-jetbrains/pull/2912

File: jetbrains-core/tst-213/software.aws.toolkits.jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt [hunk 2]
```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer
```
Comment (rli) [inline jetbrains-core/tst-213/software.aws.toolkits.jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt:4]: folder is wrong for kotlin files:
```
.../software.aws.toolkits.jetbrains/...
```

vs

```
.../software/aws/toolkits/jetbrains/...
```

there are several other files with the same problem

Comment (manodnyab) [reply jetbrains-core/tst-213/software.aws.toolkits.jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt]: moved the files to the correct path

File: jetbrains-core/tst-213/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import software.aws.toolkits.jetbrains.core.MockResourceCacheRule
+import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerNode
+import software.aws.toolkits.jetbrains.core.fillResourceCache
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import javax.swing.tree.TreeModel
+
+class AwsExplorerNodeProcessorTest {
```
Comment (kiiadi) [inline jetbrains-core/tst-213/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt:33]: why is this whole file pulled out? is this another one that could be paired down?

Comment (manodnyab) [reply jetbrains-core/tst-213/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt]: Yes, actually, the change can be gotten through a new file, added the change

**Pull request merged**